content/webapp/services/prismic/events.ts

```diff
@@ -50,18 +54,30 @@ function filterEventsByTimeRange(
 export function filterEventsForNext7Days(events: Event[]): Event[] {
   const startOfToday = london().startOf('day');
   const endOfNext7Days = startOfToday.clone().add(7, 'day').endOf('day');
-  return filterEventsByTimeRange(events, startOfToday, endOfNext7Days);
+  return filterEventsByTimeRange(
+    events,
+    startOfToday.toDate(),
+    endOfNext7Days.toDate()
+  );
 }
 
 export function filterEventsForToday(events: Event[]): Event[] {
   const startOfToday = london().startOf('day');
   const endOfToday = london().endOf('day');
-  return filterEventsByTimeRange(events, startOfToday, endOfToday);
+  return filterEventsByTimeRange(
+    events,
+    startOfToday.toDate(),
+    endOfToday.toDate()
+  );
 }
 
 export function filterEventsForWeekend(events: Event[]): Event[] {
   const { start, end } = getNextWeekendDateRange(new Date());
-  return filterEventsByTimeRange(events, london(start), london(end));
+  return filterEventsByTimeRange(
+    events,
+    london(start).toDate(),
```

Comment:
Are these Londons still needed?
getNextWeekendDateRange calculates based on London time, then returns toDate(),  and filterEventsByTimeRange looks like it doesn't use a timezone-aware range any more.

Reply:
Nope, think I had the same thought and ditched them! 😄